libs/timebar/src/timebar.js

```diff
@@ -292,6 +292,7 @@ class Timebar extends Component {
               absoluteStart={absoluteStart}
               absoluteEnd={absoluteEnd}
               onBookmarkChange={this.props.onBookmarkChange}
+              onIntervalClick={this.props.onIntervalClick}
```

Comment:
This function says in the label:
```
category: 'Timebar'
action: 'Bookmark timerange'
label: 'removed'
```
could you update it?

Reply:
this line is not needed, removed